components/ui/AppCard.vue

```diff
@@ -201,3 +207,56 @@ export default class AppCard extends Vue {
   }
 }
 </style>
+
+<style lang="scss">
+@import '~carbon-components/scss/globals/scss/typography';
+
+.app-card {
+  .bx--tag--purple {
+    background-color: $tag-background-color;
+    color: $tag-text-color;
+    margin-right: $spacing-03;
+    margin-left: 0;
+    min-width: 0;
+  }
+
+  .bx--tag--purple:last-child {
+    margin-right: 0;
+  }
+
+  &__custom-pill {
+    @include type-style('caption-01');
+
+    background-color: $tag-background-color;
+    color: $tag-text-color;
+    display: inline-flex;
+    min-width: 0;
+    max-width: 100%;
+    min-height: 1.5rem;
+    align-items: center;
+    justify-content: center;
+    padding: 0.25rem 0.5rem;
+    margin-top: 0.25rem;
+    margin-bottom: 0.25rem;
+    margin-right: $spacing-03;
+    margin-left: $spacing-03;
+    border-radius: 6.9375rem;
+    cursor: default;
+    vertical-align: middle;
+    word-break: break-word;
+
+    .bx--tooltip__trigger svg {
+      fill: white;
+      margin-left: $spacing-02;
+
+    :hover svg {
```

Comment:
Could you add the spaces/tabs needed to this block, is it inside `.bx--tooltip__trigger svg`, right?